sycl/include/CL/sycl/handler.hpp

```diff
@@ -774,6 +774,7 @@ class __SYCL_EXPORT handler {
     // 3. The kernel is provided via an interoperability method.
     // 4. The API "this_item" is used inside the kernel.
     // 5. The range is already a multiple of the rounding factor.
+    // 6. We have an ESIMD kernel
```

Comment:
It would be helpful to have a short justification why rounding is disabled for ESIMD kernels.